tree.js

```diff
@@ -0,0 +1,69 @@
+// Variables & Object Function
+
+
+var heightElement = document.getElementById("height");
+var charElement = document.getElementById("character");
+
+var tree ;
+
+function treeValues(keypress){
+
+	tree = {height: heightElement.value, character: charElement.value};
+
+    if(keypress.keyCode === 13) { 
+        
+    	
+            if (tree.height === "" || tree.character === "") {
+
+            alert("Can't build yo Tree without both parameters");
+
+    }
+    branchAdder(tree);
```

Comment:
the indentation here is confusing in particular because the ending curly brace for the internal if statement is off.